core/templates/pages/admin-page/admin-page.directive.html

```diff
@@ -11,7 +11,7 @@
   <admin-dev-mode-activities-tab ng-if="$ctrl.isActivitiesTabOpen() && $ctrl.inDevMode"
                                  set-status-message="$ctrl.setStatusMessage">
   </admin-dev-mode-activities-tab>
-  <admin-prod-mode-activities-tab ng-if="$ctrl.isActivitiesTabOpen() && !$ctrl.inDevMode">
+  <admin-prod-mode-activities-tab *ngIf="isActivitiesTabOpen() && !inDevMode">
```

Comment:
the admin-page directive is in angular so I'm not sure whether this ngIf will work here? Have you tested this change?

Reply:
Thanks for clearing, I was confused about that. Now changes are removed and it is working in my branch on  localhost.
Done.

Reply:
Done.